templates/components/form/inventory_info.html.twig

```diff
@@ -152,11 +152,17 @@
          });
       });
       </script>
+   {%  else %}
+   <div class="card-body row">
+      <div class="mb-3 col-12 col-sm-6">
+         {{ __('No agent has been linked.') }}
```

Comment:
We may have a linked agent in the past that has been removed due to stale agents configuration, or cleaning from some plugin. Maybe a vague sentence would be more appropriate.
```suggestion
         {{ __('Agent information is not available.') }}
```

Reply:
I'm not in favor of introducing a new string few days before release. Also, if we change here, it must also be changed in PHP feature file.

Reply:
Another suggestion if a shorter sentence is better.
```suggestion
         {{ __('No agent linked') }}
```